test/components/hover-card.spec.js

```diff
@@ -17,14 +18,17 @@ describe('HoverCard', () => {
 
   describe('truncateDt prop', () => {
     const Parent = {
-      template: `<hover-card icon="file" :truncate-dt="truncateDt">
+      template: `<a ref="target" href="#">Some link</a>
+      <popover :target="target">
```

Comment:
One change here is that I added a popover back to the tests. I wrote on Slack:

> In the two FIX tests, the content of the `<dd>` is just `'x'`, so the width of the `<dd>` is expected to be quite small (just large enough to fit `'x'`). But because the hover card is outside a popover, it’s growing to fit the width of the viewport. When it’s positioned absolutely, it only grows to the width it needs. So I’m thinking we can put it back in a popover or put it in another container that positions it absolutely.